packages/integration-sdk-cli/src/generator/newIntegration.ts

```diff
@@ -0,0 +1,134 @@
+import { entitiesFlow } from './entitiesFlow';
+import { relationshipsFlow } from './relationshipsFlow';
+import { configFieldsFlow } from './configFieldsFlow';
+import { stepsFlow } from './stepsFlow';
+import { generateRelationshipType, generateRelationshipName } from './helpers';
+import { yarnFormat, yarnInstall, yarnLint } from './actions';
+import { kebabCase } from 'lodash';
+import checkboxPlus from 'inquirer-checkbox-plus-prompt';
+import path from 'path';
+
+import { NodePlopAPI } from 'plop';
+import { Relationship } from './relationshipsFlow';
+import { Step } from './stepsFlow';
+
+function newIntegration(plop: NodePlopAPI) {
+  plop.setHelper('generateRelationshipType', generateRelationshipType);
+  plop.setHelper('generateRelationshipName', generateRelationshipName);
+  plop.setActionType('yarnFormat', yarnFormat);
+  plop.setActionType('yarnInstall', yarnInstall);
+  plop.setActionType('yarnLint', yarnLint);
+  plop.setPrompt('checkbox-plus', checkboxPlus);
+
+  plop.setGenerator('new:integration', {
+    description: 'Create a new integration',
+    prompts: async function (inquirer) {
+      const { vendorName } = await inquirer.prompt({
+        type: 'input',
+        name: 'vendorName',
+        message:
+          'The integration vendor name (ex. AWS, Google Workspace, Crowdstrike)',
+        validate(input) {
+          if (!input) {
+            return 'Cannot be an empty string';
+          }
+          return true;
+        },
+      });
+
+      const { packageName } = await inquirer.prompt({
+        type: 'input',
+        name: 'packageName',
+        default: `@jupiterone/graph-${kebabCase(vendorName)}`,
+        message: 'The npm package name',
+        validate(input) {
+          if (!input) {
+            return 'Cannot be an empty string';
+          }
+          return true;
+        },
+      });
+
+      const { packageDescription } = await inquirer.prompt({
+        type: 'input',
+        name: 'packageDescription',
+        message: 'A description for package',
```

Comment:
Consider a default like this: https://github.com/JupiterOne/graph-google-cloud/blob/main/package.json#L4